codeclimate-radon

```diff
@@ -2,16 +2,26 @@
 
 import json
 import os.path
+import sys
 
 threshold = "b"
+binstub = "radon3"
 include_paths = ["."]
 
 if os.path.exists("/config.json"):
     contents = open("/config.json").read()
     config = json.loads(contents)
 
-    if config.get("config") and config["config"].get("threshold"):
-        threshold = config["config"]["threshold"].lower()
+    if config.get("config"):
+        if config["config"].get("python_version"):
+            version = config["config"].get("python_version")
+            if version[0] == "2":
```

Comment:
I'm not convinced that's good behavior: if a user is specifying something that specific, it may be because there's some small difference between point releases of Python that they care about. And since we don't actually recognize or support those differences, I feel like this is would be hiding surprising behavior from their perspective.